src/infrastructure/Listener.ts

```diff
@@ -320,6 +320,18 @@ export class Listener {
         this.webSocket.send(JSON.stringify(subscriptionMessage));
     }
 
+    /**
+     * @internal
+     * @param channel - Channel to unsubscribe
+     */
+    private unsubscribeTo(channel: string) {
```

Comment:
this should be renamed to `unsubscribeFrom`

Reply:
renamed in [#559af45](https://github.com/evias/nem2-sdk-typescript-javascript/commit/559af45c598b7742952f4ca54bebde679996680c)